<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simple Paint</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

#MainWindow
{

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 224, 176, 255), stop:0.09 rgba(246, 189, 237, 255), stop:0.14 rgba(194, 207, 246, 255), stop:0.19 rgba(184, 160, 168, 255), stop:0.25 rgba(171, 186, 248, 255), stop:0.32 rgba(243, 248, 224, 255), stop:0.385 rgba(249, 162, 183, 255), stop:0.47 rgba(100, 115, 124, 255), stop:0.58 rgba(251, 205, 202, 255), stop:0.65 rgba(170, 128, 185, 255), stop:0.75 rgba(252, 222, 204, 255), stop:0.805 rgba(206, 122, 218, 255), stop:0.86 rgba(254, 223, 175, 255), stop:0.91 rgba(254, 236, 244, 255), stop:1 rgba(255, 191, 221, 255));
}
#triangleButton
{
border-image:url(:/triangle.png);
}
#rectButton
{
border-image: url(:/rect.png);
}

#elipseButton
{
border-image: url(:/round.png);
}

#clearPen
{
border-image:url(:/eraser.png);
}

#pencilButton
{
border-image: url(:/pencil.png);
}
#textButton
{
border-image: url(:/Text.png);
}
#saveButton
{
border-image: url(:/Save.png);
}

#openButton
{
border-image: url(:/open.png);
}
</string>
  </property>
  <property name="iconSize">
   <size>
    <width>40</width>
    <height>40</height>
   </size>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
min-width: 30;
max-width: 30;
min-height: 30;
max-height: 30;
border: 1 solid black;
}

#colorIndicator {
min-width: 20;
max-width: 20;
min-height: 20;
max-height: 20;
}</string>
   </property>
   <widget class="CanvasWidget" name="display" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>811</width>
      <height>481</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>161</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clearAll">
       <property name="minimumSize">
        <size>
         <width>66</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>66</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">min-width: 64;
font: 75 10pt &quot;MS Sans Serif&quot;;
max-width: 64;
background-color: rgb(232, 199, 255);</string>
       </property>
       <property name="text">
        <string>Clear all</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QSlider" name="brushSize_slider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="brushSize_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>  0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="3">
          <widget class="QPushButton" name="color_white">
           <property name="styleSheet">
            <string notr="true">
QPushButton {
	background-color: white;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    
    padding: 5px;
    }



QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">colorSelectGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="color_green">
           <property name="styleSheet">
            <string notr="true">
QPushButton {
	background-color: green;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    
    padding: 5px;
    }



QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">colorSelectGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="color_blue">
           <property name="styleSheet">
            <string notr="true">
QPushButton {
	background-color: blue;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    
    padding: 5px;
    }



QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">colorSelectGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="color_grey">
           <property name="styleSheet">
            <string notr="true">
QPushButton {
	background-color: gray;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    
    padding: 5px;
    }



QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">colorSelectGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="color_red">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: red;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    
    padding: 5px;
    }



QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">colorSelectGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="color_black">
           <property name="styleSheet">
            <string notr="true">
QPushButton {
	background-color: black;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    
    padding: 5px;
    }



QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">colorSelectGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="color_yellow">
           <property name="styleSheet">
            <string notr="true">
QPushButton {
	background-color: yellow;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    
    padding: 5px;
    }



QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">colorSelectGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="color_custom">
           <property name="styleSheet">
            <string notr="true">


QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));

    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    
    padding: 5px;
    }



QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="colorIndicator">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(166, 86, 182);

font: 75 10pt &quot;MS Sans Serif&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>10</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(28, 12, 255);
font: 75 14pt &quot;MS Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>         Tools</string>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="textButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pencilButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>33</width>
           <height>33</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearPen">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>36</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(28, 12, 255);
font: 75 14pt &quot;MS Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>          Shapes</string>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="elipseButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="triangleButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rectButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CanvasWidget</class>
   <extends>QWidget</extends>
   <header location="global">canvaswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="colorSelectGroup"/>
 </buttongroups>
</ui>
